tests/tests/test_deactivate_expired_reasons.py

```diff
@@ -0,0 +1,66 @@
+from dateutil.relativedelta import relativedelta
+
+from django.test import TestCase
+
+from freezegun import freeze_time
+from gdpr.models import LegalReason
+from germanium.tools import assert_equal, assert_false, assert_not_equal, assert_true, test_call_command
+from tests.models import Customer
+from tests.purposes import FIRST_AND_LAST_NAME_SLUG, MARKETING_SLUG
+from tests.tests.data import CUSTOMER__FIRST_NAME, CUSTOMER__KWARGS
+from tests.tests.utils import AnonymizedDataMixin
+
+
+class TestDeactivateExpiredReasons(AnonymizedDataMixin, TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.customer: Customer = Customer(**CUSTOMER__KWARGS)
+        cls.customer.save()
+
+    def test_command_should_expire_active_legal_reson(self):
+        legal_reason = LegalReason.objects.create_consent(MARKETING_SLUG, self.customer)
+        legal_reason.save()
+
+        # consent has not expired
+        test_call_command('deactivate_expired_reasons')
+        assert_true(LegalReason.objects.exists_valid_consent(MARKETING_SLUG, self.customer))
+
+        # consent has expired
+        with freeze_time(legal_reason.expires_at + relativedelta(seconds=1)):
+            test_call_command('deactivate_expired_reasons')
+        assert_false(LegalReason.objects.exists_valid_consent(MARKETING_SLUG, self.customer))
+
+    def test_command_should_not_expire_deactivated_legal_reasons(self):
```

Comment:
Maybe also test, that the command will not touch already expired consent.

Reply:
Added.